text/0517-io-os-reform.md

```diff
@@ -498,7 +500,65 @@ throughout IO, we can go on to explore the modules in detail.
 ### `std::process`
 [std::process]: #stdprocess
 
-> To be added in a follow-up PR.
+Currently `std::io::process` is used only for spawning new
+processes. The re-envisioned `std::process` will ultimately support
+inspecting currently-running processes, although this RFC does not
+propose any immediate support for doing so -- it merely future-proofs
+the module.
+
+#### `Command`
+[Command]: #command
+
+The `Command` type is a builder API for processes, and is largely in
+good shape, modulo a few tweaks:
+
+* Replace `ToCCstr` bounds with `IntoOsStrBuf`.
+* Replace `env_set_all` with `env_clear`
+* Rename `cwd` to `current_dir`, take `AsPath`.
+* Rename `spawn` to `run`
+* Move `uid` and `gid` to an extension trait in `os::unix`
+* Make `detached` take a `bool` (rather than always setting the
```

Comment:
Would be nice if this had the similar API as Thread. i.e. keep `spawn` which spawns a detached process and add `scoped` which creates a bound/attached process.